File: Python/greatest.py
li = []
for i in range(0, 10):
    num = int(input("Enter number: "))
    li.append(num)
    
# sorts = sorted(li)
# print("The Greatest among above 10 numbers are : ",sorts[-1])

max = li[0]
for i in range(0, 10):
    if li[i] > max:
        max = li[i]
        
print("The greatest among above 10 numbers are: ", max)

File: Java/greatest.java
package Java;
import java.util.Scanner;

public class greatest {
    public static void main(String args[])
    {
        int n=10;
        int[] a = new int[10];
        Scanner sc = new Scanner(System.in);
        

        for(int i=0; i<n; i++)
        {
            System.out.print("Enter the number: ");
            a[i] = sc.nextInt();
            
        }
        // System.out.println(a[0]);
        int max = a[0];

        for(int i=0; i<n; i++)
        {
            if(a[i]> max)
            {
                max = a[i];
            }
        }

        System.out.println(max);

    }
    
}
